libs/website/builder/src/lib/website-builder.context.tsx

```diff
@@ -240,6 +242,7 @@ const WebsiteBuilderContext = createContext<WebsiteBuilderProps>({
   ArticleList: NoComponent,
   CommentList: NoComponent,
   CommentListItem: NoComponent,
+  CommentListItemShare: NoComponent,
```

Comment:
I think the default provider is now missing the commentlistitemshare and wouldn't render anything